code/mob/living/carbon/human/monkeys.dm

```diff
@@ -258,6 +260,15 @@
 			return
 		..()
 		if (src.ai_state == 0)
+			if (src.equipped() && istype(src.equipped(),/obj/item/implant/projectile/body_visible/dart/bardart))
```

Comment:
```suggestion
			if (istype(src.equipped(),/obj/item/implant/projectile/body_visible/dart/bardart))
```
don't need to check it twice, `istype` will return false on `null`

Reply:
you missed this one

Reply:
no idea why i though i fixed it